cumulusci/tasks/salesforce/metadata_etl/base.py

```diff
@@ -0,0 +1,262 @@
+import tempfile
+
+from pathlib import Path
+
+from cumulusci.core.exceptions import CumulusCIException
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask, Deploy
+from cumulusci.salesforce_api.metadata import ApiRetrieveUnpackaged
+from cumulusci.tasks.metadata.package import PackageXmlGenerator
+from cumulusci.core.utils import process_bool_arg, process_list_arg
+from cumulusci.utils import elementtree_parse_file, inject_namespace
+from cumulusci.core.config import TaskConfig
+
+
+class BaseMetadataETLTask(BaseSalesforceApiTask):
+    deploy = False
+    retrieve = False
+
+    namespaces = {"sf": "http://soap.sforce.com/2006/04/metadata"}
+
+    task_options = {
+        "unmanaged": {
+            "description": "If True, changes namespace_inject to replace tokens with a blank string"
+        },
+        "namespace_inject": {
+            "description": "If set, the namespace tokens in files and filenames are replaced with the namespace's prefix"
+        },
+        "api_version": {
+            "description": "Metadata API version to use, if not project__package__api_version."
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        self.options["unmanaged"] = process_bool_arg(
+            self.options.get("unmanaged", False)
+        )
+        self.api_version = (
+            self.options.get("api_version")
+            or self.project_config.project__package__api_version
+        )
+
+    @property
+    def _namespace_injector(self):
+        return lambda text: inject_namespace(
+            "",
+            text,
+            self.options.get("namespace_inject"),
+            not self.options["unmanaged"],
+        )[1]
+
+    def _get_package_xml_content(self, deploy):
```

Comment:
I think I'm missing something here; is the `deploy` param being utilized implicitly?

Reply:
@Br4nd0R Its purpose is to allow subclasses to generate a _different_ `package.xml` for deployment than what they requested to pull.